charmcraft.yaml

```diff
@@ -72,6 +72,17 @@ requires:
 # General configuration documentation: https://juju.is/docs/sdk/config
 config:
   options:
+    external-hostname:
+      description: |
+        The DNS listing used for external connections.
+        Will default to the name of the deployed application.
+      type: string
+    
+    tls-secret-name:
+      description: |
+        Name of the k8s secret which contains the TLS certificate to be used by ingress.
+      type: string
+
```

Comment:
Comment: We could remove this entirely if we decide as a team to use the lego charm from now on. But it doesn't hurt to have.